pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/certmanager/certificate.go

```diff
@@ -49,6 +49,9 @@ const certManagerTemplate = `# The following manifests contain a self-signed iss
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
+  labels:
```

Comment:
It may be nice to also add the label `app.kubernetes.io/part-of: <operator-name>`

Reply:
@camilamacedo86 in commit https://github.com/kubernetes-sigs/kubebuilder/pull/2800/commits/30a898e5b20899d4c414a4171097a33dfd3f5d1c I added the labels `app.kubernetes.io/created-by: {{ .ProjectName }}` and `app.kubernetes.io/part-of: {{ .ProjectName }}`.

I avoided adding the `app.kubernetes.io/managed-by` label because I think once these resources are created after running `kustomize` they are managed by the cluster itself and not a controller. 

I also avoided setting the `app.kubernetes.io/version` label since I think this value should correspond to the version of the operator/project that these files are a part of since these files are parts of the larger operator project. I think if we wanted to include the `app.kubernetes.io/version` label it may be good to have it commented out with a description that if a user wants to use it, it should be set to the version of the operator/project.

WDYT?